### DISCUSSION TITLE
ESP32-S3 ILI9486 Basic problems

### DISCUSSION THREAD

**Discussion by roundspring2003** [Bodmer/TFT_eSPI#3214] [Q&A - General]
I'm trying to activate the ILI9486 with the following pins.
The following is the code you can input into the "user_setup.h" file.

#define USER_SETUP_ID 70
#define ILI9486_DRIVER
#define TFT_PARALLEL_8_BIT
// ESP32 S3 pins used for the parallel interface TFT
#define TFT_CS   4
#define TFT_DC   37  // Data Command control pin - must use a GPIO in the range 0-31
#define TFT_RST  5
#define TFT_WR   13  // Write strobe control pin - must use a GPIO in the range 0-31
#define TFT_RD   11

#define TFT_D0   2  // Must use GPIO in the range 0-31 for the data bus
#define TFT_D1   1  // so a single register write sets/clears all bits
#define TFT_D2   8
#define TFT_D3   7
#define TFT_D4   4
#define TFT_D5   5
#define TFT_D6   6
#define TFT_D7   41

#define LOAD_GLCD   // Font 1. Original Adafruit 8 pixel font needs ~1820 bytes in FLASH
#define LOAD_FONT2  // Font 2. Small 16 pixel high font, needs ~3534 bytes in FLASH, 96 characters
#define LOAD_FONT4  // Font 4. Medium 26 pixel high font, needs ~5848 bytes in FLASH, 96 characters
#define LOAD_FONT6  // Font 6. Large 48 pixel font, needs ~2666 bytes in FLASH, only characters 1234567890:-.apm
#define LOAD_FONT7  // Font 7. 7 segment 48 pixel font, needs ~2438 bytes in FLASH, only characters 1234567890:.
#define LOAD_FONT8  // Font 8. Large 75 pixel font needs ~3256 bytes in FLASH, only characters 1234567890:-.
#define LOAD_GFXFF  // FreeFonts. Include access to the 48 Adafruit_GFX free fonts FF1 to FF48 and custom fonts

#define SMOOTH_FONT

I tried running clour.test, but it only turns the screen gray. I don't know what the problem is.

**Comment by Bodmer** [comment]
The data bus must use GPIO in the range 0-31
